### PR TITLE
Run Edelweiss tests in DroneCI

### PR DESCRIPTION
**Pull request by p1gmale0n**
In a conversation with @JoranHonig , it was decided that the easiest way to get edelweiss tests back into operation - is to run them in Drone-ci.
this PR configures drone-ci to run edelweiss tests for every commit in develop and master branches and for every tag.

DroneCI uses the following special settings:
```
IGNORE_FALSE_POSITIVES=guess_the_random_number_fixed.sol simple_dao_fixed.sol simple_dao.sol old_blockhash.sol
IGNORE_REGRESSIONS=tokensalechallenge.sol gas_model_fixed.sol old_blockhash_fixed.sol dos_simple.sol dos_number.sol BECToken.sol dos_address.sol wallet_03_wrong_constructor.sol mycontract.sol```

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        from_secret: MONGO_URL
+  commands:
+    - export CIRCLE_BUILD_NUM=$DRONE_BUILD_NUMBER
+    - export CIRCLE_BUILD_URL=$DRONE_BUILD_LINK
+    - export CIRCLE_BRANCH=$DRONE_BRANCH
+    - export CIRCLE_SHA1=$DRONE_COMMIT_SHA
+    - cd /edelweiss
+    - git submodule update --init --recursive
+    - git submodule update --remote --recursive
+    - pip3 install mythril
```
Comment (JoranHonig) [inline .drone.yml:32]: won't this install the mythril ver sion from the package repo?

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - edelweiss-cli 
+      -p mythril 
+      --timeout 90 
+      --output-dir $(pwd) 
+      --s3 
+      --dynamodb 
+      --circle-ci CircleCI/mythril.csv
```
Comment (JoranHonig) [inline .drone.yml:39]: why is this tag called CircleCI?

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    AWS_DEFAULT_REGION:
+        from_secret: AWS_DEFAULT_REGION
+    AWS_ACCESS_KEY_ID:
+        from_secret: AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY:
+        from_secret: AWS_SECRET_ACCESS_KEY
+    MONGO_URL:
+        from_secret: MONGO_URL
+  commands:
+    - export CIRCLE_BUILD_NUM=$DRONE_BUILD_NUMBER
```
Comment (JoranHonig) [inline .drone.yml:25]: @p1gmale0n could you add a comment to quickly explain the reason for these exports?

File: .drone.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+steps:
+- name: edelweiss_swc
+  image: mythx.azurecr.io/tools/edelweiss
+  environment:
+    IGNORE_FALSE_POSITIVES:
+      from_secret: IGNORE_FALSE_POSITIVES
+    IGNORE_REGRESSIONS:
+      from_secret: IGNORE_REGRESSIONS
+    AWS_DEFAULT_REGION:
+        from_secret: AWS_DEFAULT_REGION
```
Comment (JoranHonig) [inline .drone.yml:17]: why is the indentation on this line different from line 15

File: .drone.yml (hunk 1 above)
Comment (p1gmale0n) [reply .drone.yml]: changed to install from local path

File: .drone.yml (hunk 3 above)
Comment (p1gmale0n) [reply .drone.yml:27]: added comments to job commands

File: .drone.yml (hunk 2 above)
Comment (p1gmale0n) [reply .drone.yml:44]: i have no idea ¯\_(ツ)_/¯

File: .drone.yml (hunk 4 above)
Comment (p1gmale0n) [reply .drone.yml]: indents fixed

**Pull request merged**